DataFormats/OnlineMetaData/interface/DCSRecord.h

```diff
@@ -68,6 +72,10 @@ class DCSRecord {
   /// Return the current of the CMS magnet in A
   float magnetCurrent() const { return magnetCurrent_; }
 
+  /// Return the magnetic field of the CMS magnet in T
+  /// The precision is 0.6 to 1.8 mT in the range of a current from 9500 to 18164 A (from Vyacheslav.Klyukhin@cern.ch)
+  float magneticField() const { return (0.0002067 * magnetCurrent_ + 0.0557973); }
```

Comment:
Where is this approximated value of the magnetic field (in the solenoid axis, I guess) expected to be used?

Reply:
Correct. I have seen a couple of instances in CMSSW code where the appropriate B-field is used. I don't recall the exact use cases. I got this formula from the CMS-magnet experts for the calculation in the OMS. I consider it useful to provide it in CMSSW, too.